<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>544</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(41, 64, 123);
	color: rgb(97, 141, 250);
}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>131</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(41, 64, 123);
	color: rgb(97, 141, 250);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo/logoPlaneta.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Автокартизатор</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="LDWindow_btn">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 2px;
	border-radius: 12px;
	border-color: #F5F5F5;
	margin-left: 20px;
	margin-right: 20px;
	margin-top: 25px;
}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
     </property>
     <property name="text">
      <string>Загрузить данные</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: transparent;
	color: #F5F5F5;
	border-style: solid;
	border-width: 2px;
	border-radius: 12px;
	border-color: #F5F5F5;
	margin-left: 20px;
	margin-right: 20px;
	margin-top: 25px;
}

QPushButton:hover{
	
	color: #618DFA;
	border-color: #618DFA;
}</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RF.qrc"/>
 </resources>
 <connections/>
</ui>
